crates/polars-time/src/windows/group_by.rs

```diff
@@ -242,7 +242,7 @@ pub(crate) fn group_by_values_iter_lookbehind(
     // Use binary search to find the initial start as that is behind.
     let mut start = if let Some(&t) = time.get(start_offset) {
         let lower = add(&offset, t, tz.as_ref())?;
-        let upper = add(&period, lower, tz.as_ref())?;
+        let upper = t; // period == -offset, so `t + offset + period` is equal to `t`
```

Comment:
Is this commented out or a comment? If a comment, can it be moved above with some additional explanation?